
/EkfCalNode:
    ros__parameters:
        Log_Level: 1

        IMU_list:
            - 'vn300'

        Camera_list:
            - 'ace'

        IMU:
            vn300:
                BaseSensor: True
                Intrinsic: False
                Rate: 400.0
                Topic: "/vectornav/imu_uncompensated"
                VarInit: [0.01, 0.01, 0.01, 0.01, 0.01, 0.01]
                PosOffInit: [0.0, 0.0, 0.0]
                AngOffInit: [1.0, 0.0, 0.0, 0.0]
                AccBiasInit: [0.0, 0.0, 0.0]
                OmgBiasInit: [0.0, 0.0, 0.0]

        Camera:
            ace:
                Rate: 5.0
                Topic:  "/my_camera/pylon_ros2_camera_node/image_raw"
                PosOffInit: [0.0, 0.0, 0.0]
                AngOffInit: [1.0, 0.0, 0.0, 0.0]
                VarInit: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1]
                Tracker: "ORB"

        Tracker:
            ORB:
                FeatureDetector: 4
                DescriptorExtractor: 0
                DescriptorMatcher: 0
                DetectorThreshold: 10.0

